.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,66 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: Please visit out slack chanel [slack](https://adminbro.slack.com/archives/C011WETJSCB) or leave your email.
+    validations:
+      required: false
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: input
+    id: prevalence
+    attributes:
+      label: Bug prevalence
+      description: "How often do you or others encounter this bug?"
+      placeholder: "Example: Whenever I visit the personal account page (1-2 times a week)"
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: AdminJS version
+      description: "on which version you encounter this bug?"
+      placeholder: "copy exact version that u are currently using"
```

Comment:
Let's not use abbreviations but full words in official communication. `u` -> `you`. Also capitalize the first letter of each sentence/statement for cohesion.